components/automate-dex/web/theme/styles.css

```diff
@@ -20,12 +20,16 @@ html, body {
     font-weight: 400;
 }
 
-button, div.dex-btn {
+button {
+    display: block;
     background-color: #3864f2;
     color: #fff;
     width: 279px;
     height: 53px;
     margin: 0 auto;
+    border-radius: 4px;
+    border: 0;
+    line-height: 33px;
```

Comment:
[nit]: looks like indentation is deep - linting will catch though

Reply:
theres no lint on the ui pages for automate-dex...

Reply:
if you by chance want to push up a commit that makes the indentation whatever the standard yall use is, that would be rad

Reply:
yeah no prob! - i'll slack before I push